src/platform/site-wide/cta-widget/index.js

```diff
@@ -437,6 +479,8 @@ const mapStateToProps = state => {
     verified,
     multifactor,
     status,
+    vaPatient,
+    mhvAccountState,
```

Comment:
@patrickvinograd I'm a little confused on the `mhvAccountState` variable, just to double check

`mhvAccountState === user['vaProfile']['mhvAccountState']`
and
`mhvAccount['accountState'] === user['profile']['mhvAccount']['accountState']`

Reply:
Oh yeah, there is a vets-api PR that goes with this 😬 that adds the va_patient and mhv_account_state to the user profile based on simple calculations of identifiers from MVI. 

The `mhvAccount['accountState']` is derived from the `v0/mhv_accounts` API endpoint and has a whole complex state machine backing it, to move the user through eligibility and upgrade calculations. My goal is to largely get rid of that - since we are pushing the account creation/upgrade logic into MHV, there's no reason for vets-api to be tracking all those states.

Reply:
Will push up that vets-api PR now.

Reply:
ok, and to double check the values of the vaProfile mhv account state are different right?  Just want to make sure that https://github.com/department-of-veterans-affairs/vets-website/pull/12624/files#diff-a8a332ec86e925e877037bbdc51e7a18R72 should not be equal to https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/validate-mhv-account/constants.js#L7

Reply:
correct...I did not re-use those values. I'm not opposed although this new attribute definitely has a subset of states compared to the mhvAccount object.